Makefile

```diff
@@ -1,40 +1,17 @@
 # Documentation
-docs/tuist/preview:
-	./make/tasks/docs/tuist/preview.sh
-docs/tuist/build:
-	./make/tasks/docs/tuist/build.sh
+docs/preview:
+	./make/tasks/docs/preview.sh
+docs/build:
+	./make/tasks/docs/build.sh
 
 # Tuist
 tuist/edit:
 	./make/tasks/tuist/edit.sh
-tuist/fetch:
-	./make/tasks/tuist/fetch.sh $(ARGS)
 tuist/generate:
 	./make/tasks/tuist/generate.sh $(ARGS)
-tuist/cache-warm:
```

Comment:
this is now part of generate

Reply:
Even though it might make sense for both to be merged when executed from a CI pipeline. It's valuable to have both actions separated locally. Indeed, I'm using `tuist/cache-warm` a lot to debug an issue at the moment.